cypress.env.json

```diff
@@ -5,5 +5,5 @@
   "emailAddress": "",
   "phoneNumber": "",
   "signalId": "",
-  "token": ""
+  "token": "eyJhbGciOiJSUzI1NiIsImtpZCI6ImUyNjQ1NWFmOThkZTI3ZTUwYTBmNDEzNjZkZjZhM2JhZWQ5OTljMzkifQ.eyJpc3MiOiJodHRwOi8vbG9jYWxob3N0OjU1NTYiLCJzdWIiOiJDZ0V4RWdWc2IyTmhiQSIsImF1ZCI6InNpZ25hbHMiLCJleHAiOjE1OTYyMTQ2MDQsImlhdCI6MTU5NjEyODIwNCwibm9uY2UiOiJZeFYvTUViNzduRnFGcmRaL1JlQ3F3PT0iLCJhdF9oYXNoIjoiUl9vcS1QZFhWdm1XRVB6RzYwdFEzZyIsImVtYWlsIjoic2lnbmFscy5hZG1pbkBleGFtcGxlLmNvbSIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJuYW1lIjoiYWRtaW4ifQ.JhJ8re_cMXMMYQSa7VXs3n700ttaFsHkhpWoV-K1CjS2ocs1mL62H321HZ1b4wMVB2g9NaqqgYC5DfVI4Izl1xpq-sUa-oBEYo7I204bqc2Wo2RWw8hhtUWUtutA-zcIa3VyaUrk3kexsCl5OoH-BNjPuYJlTm5p3_OhVDb_A4kRife-IgAWVOcsVfOvtdaTZ-fDH_yaN0n97S9jHosjp7CHoNnYEo_G3VZLNE91enV317dy-O9UO0WnTVvv3hOgYKhuXjZQwKGK1yjPjj9YSf53JFbIdShpb6D2vtcOUwi4BIIw6qStP4VfZJyEYseglp5TaVWt-5Zl-IZ5Ugqctg"
```

Comment:
Token should not be public?

Reply:
This is an arbitrary test token, but would be nice to generate this with a script. I agree.